.gitignore

```diff
@@ -5,6 +5,7 @@ runtime
 *.log
 *.csv
 *.xlsx
+*.code-workspace
```

Comment:
Think this got pushed by mistake. Please unstage this file.

Reply:
It was intentional, that is VS Code workspace files